web_src/css/repository.css

```diff
@@ -1332,6 +1332,14 @@
   left: 0;
 }
 
+.repository.branches .table a:hover {
+  text-decoration: none;
+}
+
+.repository.branches .table .ui.dropdown:hover {
+  color: var(--color-primary);
```

Comment:
Necessary or not?

Reply:
Not if we convert to proposed `btn-link` class.

Reply:
been deleted